docs/source/parquet_process.mdx

```diff
@@ -0,0 +1,235 @@
+# Query datasets from Datasets Server
+
+Datasets Server automatically converts and publishes datasets on the Hub as Parquet files.[Parquet](https://parquet.apache.org/docs/) files are column-based and they really shine when you're working with big data. There are several ways you can work with Parquet files, and this guide will show you how to:
+
+- read and query Parquet files with Pandas and Polars
+- access, read and query Parquet files with DuckDB and DuckDB-Wasm
+
+## Polars 
+
+[Polars](https://pola-rs.github.io/polars-book/user-guide/introduction.html) is a fast DataFrame library written in Rust with Arrow as its foundation.
+
+<Tip>
+
+💡 Learn more about how to get the dataset URLs in the [List Parquet files](parquet) guide.
```

Comment:
Nit: Would rename the "List Parquet files" guide from `parquet` to `parquet_list`